### DISCUSSION TITLE
Styling for :command: and :program: inline syntax

### DISCUSSION THREAD

**Discussion by tuukkamustonen** [pradyunsg/furo#260] [Ideas] [answered]
[:command:](https://www.sphinx-doc.org/en/1.4.8/markup/inline.html#role-command) and [:program:](https://www.sphinx-doc.org/en/1.4.8/markup/inline.html#role-program) render now merely as bold text:

![image](https://user-images.githubusercontent.com/94327/134636483-56b3a4c3-8b62-4166-8f6d-ecb4fe61346e.png)

Neither of these are actually intended as "invocation" like in my example there (as `tar -xzf somefile.tar.gz`). Docs say:

> :command:
>     The name of an OS-level command, such as rm.
> 
> :program:
>     The name of an executable program. This may differ from the file name for the executable for some platforms. In particular, the .exe (or other) extension should be omitted for Windows programs.

...but I doubt it would be too bad to use `:command:` like that.

How about styling at least `:command:` like `:kbd:` is now:

![image](https://user-images.githubusercontent.com/94327/134636387-b05a5ea5-2e17-48e5-979d-5e9067a9686d.png)

Or something similar?

**Comment by pradyunsg** [comment, marked as answer]
Got examples of if/how other Sphinx themes stylise this?

**Reply by tuukkamustonen** [reply]
Unfortunately no, I can't find much for it :|

Kitchen sinks seem to be missing these 99% of time.

I saw in a couple of places maybe, and it was rendered as bold.
